<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signon</class>
 <widget class="QWidget" name="Signon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>516</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>516</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background:rgb(129, 205, 255);
	font-size:15px;
	font-family:Century Gothic,sans-serif;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>471</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background:rgba(0,0,0,0.8);
	border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="set_usrname">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:rgb(25, 37, 44);
	background-color:rgb(245, 245, 245);
	font-size:16px;
	border-style:outset;
	border-radius:5px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswd_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:rgb(25, 37, 44);
	background-color:rgb(245, 245, 245);
	font-size:16px;
	border-style:outset;
	border-radius:5px;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>First Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font:11pt</string>
    </property>
    <property name="text">
     <string>   Input your Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font:11pt</string>
    </property>
    <property name="text">
     <string>   Set password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font:11pt</string>
    </property>
    <property name="text">
     <string>   Repeat the password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswd_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:rgb(25, 37, 44);
	background-color:rgb(245, 245, 245);
	font-size:16px;
	border-style:outset;
	border-radius:5px;
}</string>
    </property>
    <property name="placeholderText">
     <string>Repeat Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>351</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>351</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background:#069cfa;
	color:#fff;
	border-radius:15px;
	font: 87 12pt &quot;Arial Black&quot;;
}

QPushButton:pressed{
background-color:rgb(255, 170, 127);
border-style:inset;
}

</string>
    </property>
    <property name="text">
     <string>Send the Verification Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
font:11pt</string>
    </property>
    <property name="text">
     <string>   Set your Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nickname">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	color:rgb(25, 37, 44);
	background-color:rgb(245, 245, 245);
	font-size:16px;
	border-style:outset;
	border-radius:5px;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
